assets/pages/contributions-landing/containers/countrySwitcherHeaderContainer.js

```diff
@@ -13,7 +13,7 @@ import type { State } from '../contributionsLandingReducer';
 // ----- Setup ----- //
 
 const availableCountriesGroups: CountryGroupId[] =
-  ['GBPCountries', 'UnitedStates', 'EURCountries', 'International'];
+  ['GBPCountries', 'UnitedStates', 'EURCountries', 'International', 'Canada', 'NZDCountries'];
```

Comment:
What about the switcher on the bundles landing page?

Reply:
I have added them, good spot